lua_scripts/server/hooks/JTFServerManager.lua

```diff
@@ -0,0 +1,127 @@
+--[[
+    server hook file
+
+    maintainers Beeej aleks.kruza@gmail.com
+
+    Initial config Beeej 6/15/2020
+
+
+    This file is meant to automate JTF 191 script management. This file will load the enumerated files below in the
+    mission scripting environment
+]]
+
+dofile(lfs.writedir() .. [[Config\serverSettings.lua]])
+
+host = 'localhost'
```

Comment:
whoops, need to put a flag in here so that we can continue to have a debug link but it's ready for production.